lib/auth.js

```diff
@@ -61,7 +61,10 @@ module.exports = new Model({
   },
 
   _handleNewBearerToken: function(request) {
-    var response = JSON.parse(request.text);
```

Comment:
We should pass a single type into this function consistently. Is it JSON sometimes and not others?